src/jobs.rs

```diff
@@ -379,6 +381,16 @@ impl CommitJobInstance {
                     .arg(format!("--end-of-life={}", endoflife));
             };
 
+            if build_ref.ref_name.starts_with("app/") {
+                if let Some(endoflife_rebase) = &self.endoflife_rebase {
```

Comment:
I'm not sure this is right.
I think you need to first loop over all build_refs, find the app/* one (if any) and then generate the format and pass it to all refs. 
That way --end-of-lift-rebase=org.the.app=org.other.app willl cause org.the.app.Locale to be rebased to org.other.app.Locale, etc (build-commit-from does this already).

Reply:
I based that on this mention in `build-commit-from` man page: `Note, this is actually a prefix match, so if you say org.the.app=org.new.app, then something like org.the.app.Locale will be rebased to org.new.app.Locale.`

Reply:
Yes, that is true. However we spawn a separate build-commit-from for each ref command, and you only pass the --end-of-life-rebase to the instance that matches `app/`. Thats what i mean. You need to detect what eol argument to pass in and then pass it to *all* of the commands run.